conform-pr/main.go

```diff
@@ -87,9 +93,33 @@ type checkResult struct {
 	err   error
 }
 
-// runChecks runs all the checks on the given PR GraphQL node.
-func runChecks(ctx context.Context, action *githubactions.Action, client *graphql.Client, nodeID string) []checkResult {
-	pr := client.GetPullRequest(ctx, nodeID)
+// runChecks runs all the checks for the given PR.
```

Comment:
Let's document what it returns?

Feel freee to rephrase:
```suggestion
// runChecks runs all the checks for the given PR.
// It returns a slice of check's results and true if the PR is created by community member (false if the PR is created by a team member).
```

Reply:
Done